### ISSUE TITLE
New modules into es_extended

### ISSUE THREAD

**Issue by ahcenezdh** [esx-overextended/es_extended#39]
Hey, so it's about things we already talk on discord, to directly add into es_extended the following modules

- status
- addonaccount
- datastore
- license

**Comment by mahanmoulaei** [comment]
I've been thinking about this to see whether it's actually a good idea to do this or not. I ended up with a No!

I just changed the files structure in this [commit](https://discord.com/channels/1086795309834838087/1086795309834838092/1115889266900348990), so technically including more modules directly into the core and loading them up should become possible/easy to do if anyone really wants that.

However, "**I**" shouldn't really be including other resources into the core because:

1. The core should really reflect its name `core`, meaning only the core functions should be included in it for external resources to use.
2. Not all servers are going to use these addon resources. If they don't want a resource, they should be able to easily scrap them from the `server.cfg` without actually worrying about modifying the core to remove something(config devs or early developers)
3. Not all servers are going to have RP mode...
4. The very main reason I don't want to include them directly into the core is the lack of ability to restart them! If anything goes wrong, you gotta do a server restart or at least, the core restart, which stops a great amount of other resources...

*However, I might end up combining addoninventory, addonaccount, and datastore into 1 single resource for a more streamlined resource allocation and reduced number of server resources*
